Cargo.lock

```diff
@@ -885,17 +896,17 @@ dependencies = [
  "rand_chacha",
  "sha2 0.9.9",
  "sha3",
- "soroban-env-common 0.0.16 (git+https://github.com/stellar/rs-soroban-env?rev=422ce4110f26097e94835e0a70a4ed8c10b5bc16)",
- "soroban-native-sdk-macros 0.0.16 (git+https://github.com/stellar/rs-soroban-env?rev=422ce4110f26097e94835e0a70a4ed8c10b5bc16)",
- "soroban-wasmi",
+ "soroban-env-common 0.0.16",
+ "soroban-native-sdk-macros 0.0.16",
+ "soroban-wasmi 0.30.0-soroban (git+https://github.com/stellar/wasmi?rev=1a2bc7f)",
```

Comment:
@graydon can you think of any reason why I'd see this error? I'm thinking about setting prev to curr as well to avoid this. -
```
thread '<unnamed>' panicked at 'calculating global dep tree of Cargo.lock: Resolution("failed to find dependency: soroban-wasmi 0.30.0-soroban (git+https://github.com/stellar/wasmi?rev=1a2bc7f)")', src/rust/src/lib.rs:343:10
```